content/en/apps/guides/hosting/offline.md

```diff
@@ -0,0 +1,86 @@
+
+It is common to use [Let's Encrypt](https://en.wikipedia.org/wiki/Let%27s_encrypt) for this because they provide free certificates. Be cautioned that Let's Encrypt certificates expire after 90 days, so the server will need to be constantly updated with a new certificate.  Other CAs provide certificates that expire after a year, so this concern will always apply.
+
+After acquiring the certificate, if you are running a docker based CHT deployment, see [TLS instructions for Docker]({{< relref "/apps/guides/hosting/ssl-cert-install" >}}) to install the certificate. 
+
+### Domain Name Server
+
+In order to match the static IP of the web server to the CN in the certificate, a Domain Name Server (DNS) must be used.  This will allow any client on the LAN to easily connect to your CHT server without needing anything more than the domain name. 
+
+Most LANs will defer to the Internet Service Provider (ISP) to provide DNS, but there is no ISP in an offline scenario.  Instead, one must be provided. This DNS server will then be configured to have an `A` record (or `AAAA` in the case of IPv6) to point to the CHT server.
+
+### Dynamic Host Configuration Protocol 
+
+Any new client that connects to a network will get an IP address from a Dynamic Host Configuration Protocol (DHCP) server.  It is critical that the DHCP server for the LAN the CHT is on instructs all clients to use the DNS server configured above.  
+
+This is the final piece for any client, specifically Android clients, on the LAN to successfully connect to the CHT server and correctly use its TLS certificate.  
+
+### Wi-Fi AP 
+
+Optionally, a Wi-FI Access Point (AP) may be deployed on the LAN.  This should not be a router, but a simple AP so as  not introduce a DHCP or DNS server that could conflict with the two configured above. An AP will enable a device without Ethernet, such as an Android phone, to easily connect to the CHT server.
+
+### Expertise
+
+Deployments that don't have staff familiar with DNS, TLS Certs, DHCP, LAN topology and Linux in general should likely not use an offline CHT server. This is a complex deployment where mistakes are easy to make unless proper training is in place. 
+
+## Benefits Over Other Solutions
+
+An offline deployment may consider substituting some requirements above with these other solutions.  Note that ngrok and local-ip.co require Internet connectivity, so are not an offline solution.
+
+### ngrok
+
+Aside from the obvious benefit of not requiring Internet connectivity, when an offline solution is deployed, traffic stays 100% local.  When using either [your own reverse proxy]({{< relref "/secure-sharing-of-developer-instance" >}}) or a third party provider like [ngrok](https://ngrok.com/), traffic may traverse 100s or 1,000s of kilometers to ultimately reach the CHT server which is 10 meters away.
+
+This can help even in locations with Internet connectivity which is very slow, very expensive per megabyte, or both.
```

Comment:
Worth simplifying a bit, and avoiding the orphaned "this".

```suggestion
When an offline solution is deployed, traffic stays 100% local, whereas when using either [your own reverse proxy]({{< relref "/secure-sharing-of-developer-instance" >}}) or a third party provider like [ngrok](https://ngrok.com/), traffic may traverse 100s or 1,000s of kilometers to ultimately reach the CHT server which is 10 meters away. This can help when Internet connectivity is very slow, very expensive per megabyte, or both.
```